### DISCUSSION TITLE
Have to find out the dataset for fold 2 as it has high MAPE

### DISCUSSION THREAD

**Discussion by AM764733** [pycaret/pycaret#4010] [Q&A]
Hey Community! I am looking to get insight on my fold 2 as it has relatively higer MAPE. How can I access the data for fold 2 and predictions for it, I am assuming it has got some outlier which is hurting my metric. Any insight/support will be helpful.
![image](https://github.com/pycaret/pycaret/assets/48066620/885410ff-87a9-46d7-b194-a631e9ca60ad)

**Comment by astjohn-pilot** [comment]
I've had a need for this feature as well, but haven't found it implemented within pycaret directly. However, I have found a workaround for now by taking advantage of the argument `return_tuner=True` in the `tune_model()` method. Below is a simple code example to return the data within each fold using the OOP approach.

```python
timeseries_obj = TSForecastingExperiment()
timeseries_obj.setup(
    #setup stuff here
)
arima_mod = timeseries_obj.create_model(estimator="arima")
estimator, tuner = timeseries_obj.tune_model(estimator=arima_mod, optimize="rmse", return_tuner=True)
list(tuner.cv.split_series(timeseries_obj.X_train_transformed))
```
